detection-rules/attachment_pdf_onenote_form_submission.yml

```
@@ -0,0 +1,32 @@
+name: "Attachment: PDF with Microsoft OneNote form submission"
+description: "Detects PDF attachments containing a single sharepoint URL referencing the senders personal OneNote used for form submision lure."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and length(attachments) > 1
```

```suggestion
  and length(filter(attachments, .file_type == "pdf")) >= 1
```
For your consideration... something like this might improve performance of the rule.. I'm still running hunts and it's taking forever, so it might not (lol) but just a thought.

---

Thanks for this! I need to remember using filter